packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -61,11 +83,16 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
             collapsedNodes: [],
             orderedNodes: [],
             xyData: {},
-            columns: [{title: 'Name', sortable: true}]
+            columns: [{title: 'Name', sortable: true}],
+            zoomSelectionRangeStart: 0,
+            zoomSelectionRangeEnd: 0,
         };
 
         this.afterChartDraw = this.afterChartDraw.bind(this);
         this.lineChartRef = React.createRef();
+        this.instanceId = XYOutputComponent.instanceCounter++;
+        signalManager().on(Signals.TIMEGRAPH_ZOOMED, this.onTimeGraphZoomed);
+        signalManager().on(Signals.TIMEGRAPH_RESET, this.onTimeGraphReset);
```

Comment:
Can we listen to these signals in the parent class?

Reply:
I tried and I could not put the signals in AbstractTreeOutputComponent